test/analytics/tests/instrumented_events/from_the_server/core_context_providers.ts

```diff
@@ -14,12 +14,16 @@ export default function ({ getService }: FtrProviderContext) {
   const deployment = getService('deployment');
   const ebtServerHelper = getService('kibana_ebt_server');
 
-  // Failing: See https://github.com/elastic/kibana/issues/132931
-  describe.skip('Core Context Providers', () => {
+  describe('Core Context Providers', () => {
     let event: Event;
     before(async () => {
-      // Wait for the 2nd "status_changed" event. At that point all the context providers should be set up.
-      [, event] = await ebtServerHelper.getLastEvents(2, ['core-overall_status_changed']);
+      let i = 2;
+      do {
+        // Wait until we get a GREEN "status_changed" event. At that point all the context providers should be set up.
+        const events = await ebtServerHelper.getLastEvents(i, ['core-overall_status_changed']);
+        event = events[i - 1];
+        i++;
+      } while (event.properties.overall_status_level !== 'available');
```

Comment:
What are we trying to do here with that manually maintained `i` index? If we're expecting/waiting for emissions of `core-overall_status_changed` events until one is `available`, can't we just take / look at the last one? Or did I miss something :) ?

Reply:
That was my intention when I implemented `getLastEvents`. However, I noticed that it actually returns the first N events, and I couldn't find a way to change the RxJS observables to return the actual last events. 

We should probably rename the helper for now to avoid confusion. What do you think?

Reply:
Yea, the function name is misleading given what you describe, so we should probably rename it!

Reply:
https://github.com/elastic/kibana/issues/133091